### PR TITLE
yolo2

### PR DESCRIPTION
**Pull request by maximebedard**

### CODE REVIEW

File: config.ru [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 require ::File.expand_path("../config/environment", __FILE__)
 run Rails.application
+yolo
```
Comment (maximebedard) [inline config.ru:5]: @maximebedard shipit

**Comment by maximebedard** [comment]
yolo

**Comment by maximebedard** [comment]
wefwefwef

**Comment by maximebedard** [comment]
@maximebedard

**Comment by JakeCataford** [comment]
shipit

**Comment by maximebedard** [comment]
qwdqwdqwd

**Comment by JakeCataford** [comment]
can you take a look at this?

**Comment by JakeCataford** [comment]
shipit

**Comment by JakeCataford** [comment]
@maximebedard

**Comment by JakeCataford** [comment]
@maximebedard

**Pull request closed without merging**